lib/domain/models/occurrence_model.dart

```diff
@@ -16,7 +17,7 @@ abstract class OccurrenceModel
   int get code;
   @nullable
   String get updatedAt;
-
+  LatLng get coors;
```

Comment:
What is `coors`?

Reply:
Is coordinates. Do you think that is better change to coordinates?

Reply:
yes please 👍